internal-packages/next-dev/package.json

```diff
@@ -20,7 +20,7 @@
 		"devBindingsOptions.ts"
 	],
 	"dependencies": {
-		"miniflare": "^3.20231030.2",
+		"miniflare": "3.20231002.0",
```

Comment:
- [ ] Create a minimal reproduction of the regression and open a new issue regarding this in workers-sdk

Reply:
PS: I did try to create a minimal Miniflare reproduction by using standard DOs there but I couldn't really reproduce this, so I _think_ that this is related to both Miniflare and the external DOs worker we generate here 🤔 😓

Reply:
@mrbbot this [demo/test app](https://github.com/dario-piotrowicz/next-on-pages-stable-next-dev-test-demo) renders the following home page:
![Screenshot 2024-01-08 at 12 19 31](https://github.com/cloudflare/next-on-pages/assets/61631103/553bc567-b8fb-47e7-b057-e9dbd0e3c65b)

If I however use the latest miniflare version this is what I get instead:
![Screenshot 2024-01-08 at 12 18 39](https://github.com/cloudflare/next-on-pages/assets/61631103/3d9cacee-3251-464d-9dfc-981b219f1476)

I guess that it might be coming from here:
https://github.com/cloudflare/next-on-pages/blob/f0df948a8ed3df15f20259a45b5d60ddb919dfa8/internal-packages/next-dev/src/wrangler.ts#L68

I'm not sure why this is anyways, but it does indeed only happen with the latest miniflare versions 😕 

(Also I just noticed that the var is not showing up! 😮)

Reply:
I've created a new issue in the next-on-pages repo to keep track of this: https://github.com/cloudflare/next-on-pages/issues/621